hathor/indexes/tokens_index.py

```diff
@@ -115,7 +116,12 @@ def get_token_info(self, token_uid: bytes) -> TokenIndexInfo:
         raise NotImplementedError
 
     @abstractmethod
-    def create_token_info(self, token_uid: bytes, name: str, symbol: str, total: int = 0) -> None:
+    def create_token_info(self,
+                          token_uid: bytes,
+                          name: str,
+                          symbol: str,
+                          token_info_version: Optional[TokenInfoVersion],
```

Comment:
It should never be `None`; and it's the caller responsibility to pass the correct token version.